modules/cpmso-openstack-ts-root-volume-azs.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control_plane_machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-openstack-ts-root-volume-azs_{context}"]
+= Configuring {rh-openstack} clusters that have machines with root volume availability zones after an upgrade
+
+For some clusters that run on {rh-openstack-first} that you upgrade, you must manually update machine resources before you can use control plane machine sets if the following configurations are true:
+
+* You upgraded the cluster from {product-title} 4.13 to 4.14.
+ 
+* The cluster infrastructure is installer-provisioned.
+
+* Machines were distributed across multiple availability zones.
+
+* Machines were configured to use root volumes for which block storage availability zones were not defined.
+
+To understand why this procedure is necessary, see link:https://access.redhat.com/solutions/7013893[Solution #7024383].
```

Comment:
We need a release note for this. We have our first customer case now, they didn't follow the exact procedure.

Reply:
Also looks like a copy-paste error in that link.